zerver/views/user_groups.py

```diff
@@ -98,8 +99,9 @@ def edit_user_group(
     user_group_id: int = REQ(json_validator=check_int, path_only=True),
     name: Optional[str] = REQ(default=None),
     description: Optional[str] = REQ(default=None),
+    can_mention_group_id: Optional[int] = REQ(json_validator=check_int, default=None),
 ) -> HttpResponse:
-    if name is None and description is None:
+    if name is None and description is None and can_mention_group_id is None:
         raise JsonableError(_("No new data supplied"))
```

Comment:
I think we've been moving away from having checks like this in API endpoints; we should discuss removing this check as a potential API change to this endpoint in "api design".

Reply:
Started discussion - https://chat.zulip.org/#narrow/stream/378-api-design/topic/PATCH.20.2Fuser_groups.2F.7Buser_group_id.7D